tests/Behat/Services/Provider/MessagesProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Services\Provider;
+
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+use Webmozart\Assert\Assert;
+
+final class MessagesProvider
+{
+    /** @var string */
+    private $spoolDirectory;
+
+    public function __construct(string $spoolDirectory)
+    {
+        $this->spoolDirectory = $spoolDirectory;
+    }
+
+    /**
+     * @return array|\Swift_Message[]
+     */
+    public function getMessages(): array
+    {
+        $finder = new Finder();
+        $directory = $this->spoolDirectory;
+        $finder->files()->name('*.message')->in($directory);
+        Assert::notEq($finder->count(), 0, sprintf('No message files found in %s.', $directory));
```

Comment:
We could easily allow for an empty array.

Reply:
Indeed, I also see no problem with that